awx/main/models/notifications.py

```diff
@@ -68,6 +70,42 @@ class Meta:
 
     notification_configuration = JSONField(blank=False)
 
+    messages = JSONField(
+        null=True,
+        default={'started': { 'message': DEFAULT_MSG, 'body': ''},
+                 'success': { 'message': DEFAULT_MSG, 'body': ''},
+                 'error': { 'message': DEFAULT_MSG, 'body': ''}},
+        help_text=_('Optional custom messages for notification template.'))  # TODO: i18n?
```

Comment:
> # TODO: i18n?

What is this todo for? The default message doesn't need to be translated... wrapping this in `_(...)` will automatically mark it for translation.

Reply:
@matburt - the todo here was mostly for myself; I didn't know how we handled i18n (but recognized `_()` as part of internationalization). Wanted to make sure that if there was something I needed to do here that I didn't forget. Good to know that `_(...)` will automatically mark strings for expressions. 👍